src/typhoon_automator/automator.py

```diff
@@ -213,8 +223,48 @@ def save_scenarios(
     def run(
             self,
             use_vhil: bool = False):
+        """ Run all scenarios in the automation
+
+        :param bool use_vhil:
+        """
+        
         self._model.load_to_setup(use_vhil=use_vhil)
-        raise NotImplementedError()
+        
+        if self._orchestrator is None:
```

Comment:
This error check could be moved a few lines earlier, before 'load_to_setup', just to save the user some time (i.e. if it's going to fail because _orchestrator is None, don't take the time to load the stuff to Typhoon)